non-critical-infra/packages/hydra-queue-runner/default.nix

```diff
@@ -1,102 +0,0 @@
-{
```

Comment:
Might want to upstream this as it has the withOtel option and separate packages for the builder and runner.

Reply:
@Conni2461 Can you pick this?